### PR TITLE
Clear cookies when revoking an authorization

### PR DESCRIPTION
**Pull request by samueltardieu**
Suggested in #6269.

### CODE REVIEW

**Comment by Lineflyer** [comment]
Which user action triggers the mentioned "revoking of authorization"?

**Comment by pstorch** [comment]
It's triggerd by long press on `Authorize..' or your avatar.

File: main/src/cgeo/geocaching/settings/CredentialsPreference.java [hunk 1]
```diff
@@ -135,6 +136,9 @@ protected boolean isAuthorized() {
 
     @Override
     protected void revokeAuthorization() {
+        if (credentialsMapping == CredentialActivityMapping.GEOCACHING) {
+            Cookies.clearCookies();
```
Comment (Bananeweizen) [inline main/src/cgeo/geocaching/settings/CredentialsPreference.java]: Just wondering if the clearing has any negative effect on the other authorizations? But generally I'm completely fine with the change.

Comment (samueltardieu) [inline main/src/cgeo/geocaching/settings/CredentialsPreference.java]: It shouldn't. Cookies store session information for sites not using OAuth. Clearing the cookies will go through the classical login phase (a few extra requests) when data is next requested, but no interaction will need to take place.

**Pull request merged**